onos-config/templates/deployment.yaml

```diff
@@ -165,9 +165,11 @@ spec:
           imagePullPolicy: {{ $root.Values.image.pullPolicy }}
           args:
             - "{{ .port }}"
+          {{ - if .prefixed }}
           env:
             - name: PREFIXED
-              value: {{ .prefixed | quote }}
+              value: { { .prefixed | quote } }
```

Comment:
Inconsistent formatting: remove the spaces between the braces (GoLand does this to me)